doc/cascadia/Cascading-Default-Settings.md

```diff
@@ -458,6 +535,15 @@ to their modifications.
   behave correctly. It's possible that we could abstract our implementation into
   a WinRT interface that extensions could implement, and be triggered just like
   other dynamic profile generators.
+  - Additionally, we could expand the dynamic profile generator contract to

```

Comment:
Now here is a thought. If we _force_ the GUID, we can encode `source: '{guid-of-source}'` in the user copy of the profile. If a profile has a `source`, but that source didn't early-generate a profile of the same guid, THEN we just fail to apply. We don't need any logic to say "if the command line is blank, skip it"!